src/caches/rosenbrock_caches.jl

```diff
@@ -73,7 +73,9 @@ function alg_cache(alg::Rosenbrock23,u,rate_prototype,::Type{uEltypeNoUnits},::T
   tf = TimeGradientWrapper(f,uprev,p)
   uf = UJacobianWrapper(f,t,p)
   linsolve_tmp = zero(rate_prototype)
-  linsolve = alg.linsolve(Val{:init},uf,u)
+
+  linprob = LinearProblem(W,vec(u); u0=vec(u))
```

Comment:
need to do this to the whole file, and the whole extrapolation caches file

Reply:
sounds cumbersome. vim macros are might speed it up